src/openloco/ui/dropdown.cpp

```diff
@@ -78,6 +78,11 @@ namespace openloco::ui::dropdown
         add(static_cast<uint8_t>(index), title, { l });
     }
 
+    int16_t get_argument_at(size_t index, uint8_t offset)
+    {
+        return *(int16_t*)&_dropdownItemArgs[index][offset * 2];
```

Comment:
I think this is an ugly hack. Perhaps it's better to keep a local copy for the cargo dropdown.